### PR TITLE
csredis引用版本升级到最新版, 替换必要的类型转换double->decimal

### PR DESCRIPTION
**Pull request by MartinYl**
因为csredis在[v3.3.0版本](https://github.com/2881099/csredis/issues/226#issuecomment-568838238)中把double转为decimal, 所以升级csredis引用, 就会涉及到类型修改, 为了兼容, 只修改了必要的接口类型, 其他都在csredis实现中做类型转换.

建议所有类型都做修改, 但是会涉及到`StackExchange.Redis`的实现方法, 请慎重考虑

### CODE REVIEW

**Comment by MartinYl** [comment]
`EasyCaching.Redis.DefaultRedisCachingProvider.GeoPos(string cacheKey, List<string> members)`
StackExchange.Redis使用double类型，导致损失精度，测试无法通过
`13.361389338970184：StackExchange.Redis`
`13.3613893389702：StackExchange.Redis(double->decimal)`
`13.36138933897018433：csredis`

**Comment by MartinYl** [comment]
#254

File: src/EasyCaching.Core/IRedisCachingProvider.cs [hunk 1]
```diff
@@ -916,14 +916,14 @@ public interface IRedisCachingProvider
         /// <param name="cacheKey"></param>
         /// <param name="members"></param>
         /// <returns></returns>
-        List<(double longitude, double latitude)?> GeoPos(string cacheKey, List<string> members);
+        List<(decimal longitude, decimal latitude)?> GeoPos(string cacheKey, List<string> members);
```
Comment (catcherwong) [inline src/EasyCaching.Core/IRedisCachingProvider.cs:919]: Maybe it should be better to maintain the definition of the interface and transform it in the implementation of csredis.

If someone is using the method, it‘s not a good way to modify this one.

Comment (catcherwong) [reply src/EasyCaching.Core/IRedisCachingProvider.cs:919]: There are some difference between se.redis and cs.redis. we should be united here.

**Review by catcherwong — CHANGES_REQUESTED** [review]
Change the target branch from `master` to `dev`

File: src/EasyCaching.Core/IRedisCachingProvider.cs (hunk 1 above)
Comment (MartinYl) [reply src/EasyCaching.Core/IRedisCachingProvider.cs:919]: Yes, for compatibility, the interface definition should not be modified.
However, double does lose precision, refer to [#csredis/issues/226](https://github.com/2881099/csredis/issues/226), so the perfect solution should be to upgrade all double types to decimal, improve the precision, consider putting it in the next major version.

Comment (catcherwong) [reply src/EasyCaching.Core/IRedisCachingProvider.cs:919]: All right, v1.x will follow it.

**Review by catcherwong — APPROVED** [review]

**Pull request merged**